CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.h

```diff
@@ -55,8 +55,8 @@ class CaloTPGTranscoderULUT : public CaloTPGTranscoder {
   // Member Variables
   double nominal_gain_;
   double lsb_factor_;
-  int rct_factor_;
-  int nct_factor_;
+  double rct_factor_;
+  double nct_factor_;
```

Comment:
@akhukhun - this is a naively surprising change - what is the logic behind this?